### PR TITLE
Feature/next/cypress walkthroughs

### PR DESCRIPTION
**Pull request by ashblue**

### CODE REVIEW

File: cypress/integration/homepage.js [hunk 1]
```diff
@@ -14,4 +14,34 @@ describe('Homepage', () => {
         expect(value.find('a').attr('href')).to.include('/locations/');
       });
   });
+
+  it('should open an article when clicking it', () => {
+    cy.get('.grav-c-card-basic')
+      .find('a')
+      .first()
+      .click();
```
Comment (james-nash) [inline cypress/integration/homepage.js]: Question: Does this not need some kind of assertion at the end?

Comment (dw-buildit) [reply cypress/integration/homepage.js]: Yes, I think there should be something to assert that the page exists and we've navigated to it.

**Comment by ashblue** [comment]
After a lot of googling I was able to figure out how to store a piece of text with Cypress (was surprisingly difficult). This should be good to ship.

**Comment by dw-buildit** [comment]
Nice work

**Pull request merged**